exporters/otlp/trace/src/test/java/io/opentelemetry/exporter/otlp/trace/OtlpGrpcSpanExporterTest.java

```diff
@@ -135,13 +139,27 @@ void testExport_MultipleSpans() {
         OtlpGrpcSpanExporter.builder().setChannel(inProcessChannel).build();
     try {
       assertThat(exporter.export(spans).isSuccess()).isTrue();
-      assertThat(fakeCollector.getReceivedSpans())
-          .isEqualTo(SpanAdapter.toProtoResourceSpans(spans));
+      assertThat(fakeCollector.getReceivedSpans()).isEqualTo(toResourceSpans(spans));
     } finally {
       exporter.shutdown();
     }
   }
 
+  private static List<ResourceSpans> toResourceSpans(List<SpanData> spans) {
+    return Arrays.stream(ResourceSpansMarshaler.create(spans))
```

Comment:
NIT: this can be extracted into a test util class as duplicated in two places.

Reply:
They are two different Gradle projects so it's not so simple. If we had an unpublished project like testing-internal I could put it in there but for now think a copy is fine.